core/common/src/main/java/alluxio/heartbeat/ScheduledTimer.java

```diff
@@ -46,9 +48,12 @@ public final class ScheduledTimer implements HeartbeatTimer {
    * Creates a new instance of {@link ScheduledTimer}.
    *
    * @param threadName the thread name
-   * @param intervalMs the heartbeat interval (unused)
+   * @param clock for telling the current time (unused)
+   * @param intervalSupplier Sleep time between different heartbeat supplier (unused)
+   * @param periodCronExpressionSupplier the period cron expression (unused)
    */
-  public ScheduledTimer(String threadName, long intervalMs) {
+  public ScheduledTimer(String threadName, Clock clock, Supplier<Long> intervalSupplier,
+      Supplier<String> periodCronExpressionSupplier) {
```

Comment:
Can use the new interval supplier interface here

Reply:
Yeah